Base/res/js/Spreadsheet/runtime.js

```diff
@@ -162,6 +162,9 @@ class Ranges {
 
 class Range {
     constructor(startingColumnName, endingColumnName, startingRow, endingRow, columnStep, rowStep) {
+        // have to account for '0' since js will parse `+'0'` to 0
+        if (columnStep === 0 || columnStep === '0' || rowStep === 0 || rowStep === '0')
```

Comment:
I think you can also do
`if (columnStep == '0' || rowStep == '0')` if you want to have it shorter, but I guess strict checks are nicer

Reply:
yep, done